pom.xml

```diff
@@ -29,7 +29,7 @@
     <data.pipeline.parent>system:cdap-data-pipeline[6.1.2,7.0.0-SNAPSHOT)</data.pipeline.parent>
```

Comment:
Make this also 6.8.0-SNAPSHOT,7.0.0-SNAPSHOT

Reply:
Done